### PR TITLE
ext: update ci specs

### PR DESCRIPTION
**Pull request by drodriguezhdez**

### CODE REVIEW

File: lib/ddtrace/ext/ci.rb [hunk 1]
```diff
@@ -63,24 +63,32 @@ def filter_sensitive_info(url)
...
+          commit = env['APPVEYOR_REPO_COMMIT']
+          branch = (env['APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH'] || env['APPVEYOR_REPO_BRANCH'])
+          tag = env['APPVEYOR_REPO_TAG_NAME']
+        end
+
         {
           TAG_PROVIDER_NAME => 'appveyor',
-          Git::TAG_REPOSITORY_URL =>  "https://github.com/#{env['APPVEYOR_REPO_NAME']}.git",
-          Git::TAG_COMMIT_SHA => env['APPVEYOR_REPO_COMMIT'],
+          Git::TAG_REPOSITORY_URL =>  repository,
```
Comment (marcotc) [inline lib/ddtrace/ext/ci.rb:76]: If the `if` statement above is not executed, do we expect `Git::TAG_REPOSITORY_URL` to be empty?

File: lib/ddtrace/ext/ci.rb [hunk 2]
```diff
@@ -63,24 +63,32 @@ def filter_sensitive_info(url)
 
       def extract_appveyor(env)
         url = "https://ci.appveyor.com/project/#{env['APPVEYOR_REPO_NAME']}/builds/#{env['APPVEYOR_BUILD_ID']}"
+
+        if !env['APPVEYOR_REPO_PROVIDER'].nil? && env['APPVEYOR_REPO_PROVIDER'] == 'github'
```
Comment (marcotc) [inline lib/ddtrace/ext/ci.rb:67]: There's no need to nil check here, as `nil == 'github'` returns false:
```suggestion
        if env['APPVEYOR_REPO_PROVIDER'] == 'github'
```

File: lib/ddtrace/ext/ci.rb (hunk 1 above)
Comment (drodriguezhdez) [reply lib/ddtrace/ext/ci.rb:76]: Yes, we're supporting only Github as SCM tool atm, but you're right. We will iterate this point in the future for sure.

**Review by marcotc — APPROVED** [review]
GitHub is having a hard time with the "Files changed" here, but a local `git diff` looks all good.

**Pull request merged**